api/pkg/apis/projectcalico/v3/bgppeer.go

```diff
@@ -100,6 +100,10 @@ type BGPPeerSpec struct {
 	// This removes BGP loop prevention and should only be used if absolutely necesssary.
 	// +optional
 	NumAllowedLocalASNumbers *int32 `json:"numAllowedLocalASNumbers,omitempty"`
+	// TTLSecurity enables the generalized TTL security mechanism (GTSM). The provided value is the number of hops
+	// (edges) between the peers.
+	// +optional
+	TTLSecurity *uint32 `json:"ttlSecurity,omitempty"`
```

Comment:
Do we need any validation on this field? What is the maximum allowed value? Will BIRD be OK with a value of max(uint32)?

Reply:
Adding to this, both TTL in IPv4 and hop limit in IPv6 are 8 bit fields, so technically their maximum value is 255. Would it be possible and/or worth it to use `*uint8` here instead?